packages/rainbowkit/src/wallets/getDefaultWallets.ts

```diff
@@ -25,13 +21,12 @@ export const getDefaultWallets = ({
     {
       groupName: 'Popular',
       wallets: [
-        injectedWallet({ chains }),
```

Comment:
I had to get rid of this `injectedWallet` for now. Since we can't see the connector information such as `id`, `name` and etc. It's only visible after `createConfig` from wagmi.